src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="product">
+      <div class="product__card" data-qa="card">
+
+        <img class="picture" src="images/imac.jpeg"
+        alt="imac">
+
+        <div class="product__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product__code">
+          Product code: 195434
+        </div>
+
+        <div class="product__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="reviews">Reviews: 5</div>
+        </div>
+
+        <div class="product__price">
+
+          <div class="price">Price:</div>
+
+          <div class="sum">$2,199</div>
+        </div>
+
+        <buttom class="product__buttom">
+          <a class="link" href="#product"
+          data-qa="hover">Buy</a>
+        </buttom>
```

Comment:
```suggestion
        <button class="product__buttom">
          <a class="link" href="#product"
          data-qa="hover">Buy</a>
        </button>
```
Did you mean button? 😄